nvflare/lighter/impl/he.py

```diff
@@ -27,6 +27,15 @@ def __init__(
         scale_bits=40,
         scheme="CKKS",
     ):
+        """Generates Tenseal homomorphic encryption context for server and client and writes them to server and client
+        participant folders.
+
+        Args:
+            poly_modulus_degree: defaults to 8192.
+            coeff_mod_bit_sizes: defaults to [60, 40, 40].
+            scale_bits: defaults to 40.
+            scheme: defaults to "CKKS".
```

Comment:
What does each of these arguments mean?

@holgerroth can you help?

Thanks.

Reply:
I think those parameters are from tenseal package.  The parameter names should be identical to tenseal.  The actual meanings might be somewhere in the paper.  It should be safe to assume users who would like to change those parameters have already read the paper.